src/workerd/api/hyperdrive.c++

```diff
@@ -80,7 +80,7 @@ uint16_t Hyperdrive::getPort() {
 
 kj::String Hyperdrive::getConnectionString() {
   return kj::str(getScheme(), "://", getUser(), ":", getPassword(), "@", getHost(), ":", getPort(),
-      "/", getDatabase(), "?sslmode=prefer");
```

Comment:
you might consider putting the change behind an autogate so that the availability of the change can be rolled out with a salt gate instead of needing a new release.

Reply:
I'll have to explore that, I haven't spent a ton of time in the core edgeworker repos yet. Thanks for the tip!

I think this particular change should be okay going out with the next release, though.

Reply:
Good to patch this now and have a proper fix in the next release, customers have to install specific version. All local development is broken right now